cfme/configure/configuration/__init__.py

```diff
@@ -1794,12 +1485,12 @@ def _form_mapping(self, create=None, **kwargs):
         }
 
     def create(self):
-        sel.force_navigate("cfg_settings_region_my_company_tag_new", context={"tag": self})
+        navigate_to(self, 'Add')
+        sel.click(tag_form.new)
```

Comment:
No, I think this is a right one, adding tags is a bit funny, the add page is also the list page I think iirc